### PR TITLE
Fake object recognition to develop

### PR DESCRIPTION
**Pull request by hwiedDFKI**

### CODE REVIEW

File: sim/src/core/SimEntity.cpp [hunk 1]
```diff
@@ -104,6 +105,25 @@ namespace mars {
       }
     }
 
+    long unsigned int SimEntity::getRootestId(std::string name_specifier /*="" */) {
+      unsigned int id_specified = 10000;
+      unsigned int id_lowest = 10000;
```
[the remark this answers is not in the document]
Comment (hwiedDFKI) [reply sim/src/core/SimEntity.cpp]: Fixed by returning INVALID_ID if the search fails

[the remark this answers is not in the document]
Comment (malter) [reply sim/src/core/SimEntity.cpp]: You could also initialize the ids with INVALID_ID?

[the remark this answers is not in the document]
Comment (hwiedDFKI) [reply sim/src/core/SimEntity.cpp]: Done!

[the remark this answers is not in the document]
Comment (malter) [reply sim/src/core/SimEntity.cpp]: Did you updated the request?

**Pull request merged**